pkg/github/pull_request_reviews.go

```diff
@@ -0,0 +1,244 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/grafana/github-datasource/pkg/models"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+)
+
+// QueryListPullRequests lists all pull requests in a repository
+//
+//			{
+//			  search(query: "is:pr repo:grafana/grafana merged:2020-08-19..*", type: ISSUE, first: 100) {
+//			    nodes {
+//			      ... on PullRequest {
+//	              Number
+//	              Title
+//	              URL
+//	              State
+//	              Author
+//	              Repository
+//				     reviews(first: 100) {
+//			           createdAt
+//			           updatedAt
+//			           state
+//			           url
+//			           author {
+//			             id
+//			             login
+//			             name
+//			             company
+//			             email
+//			             url
+//			           }
+//			           comments(first: 0) {
+//			             totalCount
+//		               }
+//				     }
+//				  }
+//				}
+//			  }
+//			}
+type QueryListPullRequestReviews struct {
+	Search struct {
+		Nodes []struct {
+			PullRequest struct {
+				Number     int64
+				Title      string
+				URL        string
+				State      githubv4.PullRequestState
+				Author     Author
+				Repository Repository
+				Reviews    struct {
+					Nodes []struct {
+						Review struct {
+							CreatedAt githubv4.DateTime
+							UpdatedAt githubv4.DateTime
+							URL       string
+							Author    Author
+							State     githubv4.PullRequestReviewState
+							Comments  struct {
+								TotalCount int64
+							} `graphql:"comments(first: 0)"`
+						} `graphql:"... on PullRequestReview"`
+					}
+					PageInfo models.PageInfo
+				} `graphql:"reviews(first: 100, after: $reviewCursor)"`
+			} `graphql:"... on PullRequest"`
+		}
+		PageInfo models.PageInfo
+	} `graphql:"search(query: $query, type: ISSUE, first: 100, after: $prCursor)"`
+}
+
+type Author struct {
+	User models.User `graphql:"... on User"`
+}
+
+type Review struct {
+	CreatedAt     githubv4.DateTime
+	UpdatedAt     githubv4.DateTime
+	URL           string
+	Author        Author
+	State         githubv4.PullRequestReviewState
+	CommentsCount int64
```

Comment:
Just keeping track of the number of comments; otherwise, this gets even more nested with PRs containing Reviews containing Comments